pkg/gds/admin.go

```diff
@@ -149,22 +160,193 @@ func (s *Admin) setupRoutes() (err error) {
 	return nil
 }
 
+// Set the maximum age of authentication protection cookies.
+const protectAuthenticateMaxAge = time.Minute * 10
```

Comment:
The time that we allow people to finish logging in - open to discussion.

Reply:
I don't think there is a convention for this; or at least I haven't found one. 10 min is as good a guess as any I suppose.